tool/site/client/tests/pages/__snapshots__/sendPage.spec.tsx.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`Displays confirmation page if multiple prescriptions are sent successfully 1`] = `
 "<h1 class=\\"nhsuk-label-wrapper\\"><label class=\\"nhsuk-label nhsuk-label--xl\\">Send Results</label></h1>
-<div class=\\"sc-dkPtRN jXpeDI\\"><button class=\\"nhsuk-button\\" aria-disabled=\\"false\\" type=\\"submit\\">Copy Prescription IDs</button><a class=\\"nhsuk-button\\" role=\\"button\\" aria-disabled=\\"false\\" draggable=\\"false\\" href=\\"baseUrl/download/exception-report\\">Download Exception Report</a></div>
+<div class=\\"sc-dkPtRN jXpeDI\\"><button class=\\"nhsuk-button\\" aria-disabled=\\"false\\" type=\\"submit\\">Copy Prescription IDs</button></div>
```

Comment:
wheres the exception report gone?

Reply:
all prescriptions are succesful so the report is empty and we do not want to show it